requirements.txt

```diff
@@ -2,7 +2,7 @@
 
 BeautifulSoup4
 werkzeug
-Flask>=0.10
+Flask>=0.10,<2
```

Comment:
Come on, give me some credit, I can fix that! 🤩

Reply:
Hehe, I did't see your solution until now. I'll revert and look at that first. Also, this didn't go all the way anyway. 😆